.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env
+.env.development.local
```

Comment:
these files dont exist please remove from git ignore

Reply:
if someone is working on mac platform or add these files in future. So best practice is to add these files in gitignore. 

Its standard practice